src/views/Login/login.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import loginAction from "../../redux/actions/loginAction";
+import Button from "../Button";
+import LoginForm from "./LoginForm";
+import SocialAuth from "./socialAuth";
+
+class LoginPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { email: "", password: "" };
+    this.handleChange = this.handleChange.bind(this);
+    this.onLogin = this.onLogin.bind(this);
+  }
+
+  onLogin(event) {
```

Comment:
Use the same link and feedback I gave you up there to refactor this as well, remove all bind statements.